doc/examples/xx_applications/plot_thresholding.py

```diff
@@ -0,0 +1,261 @@
+"""
+============
+Thresholding
+============
+
+Thresholding is used to create a binary image from a grayscale image [1]_.
+It is the simplest way to segment objects from a background.
+
+Thresholding algorithms implemented in scikit-image can be separated in two
+categories:
+
+- Histogram-based. The histogram of the pixel intensity is used and
+  assumptions may be made on the properties of this histogram (e.g. bimodal).
+- Local. To process a pixel, only the neighboring pixels are used.
+  These algorithms often require more computation time.
+
+
+If you are not familiar with the details of the different algorithms and the
+underlying assumptions, it is often to know which algorithm will give the best
+results. Therefore, Scikit-image includes a function to test thresholding
+algorithms provided in the library. At a glance, you can select the best
+algorithm for you data, without a deep understanding of their mechanisms.
+
+.. [1] https://en.wikipedia.org/wiki/Thresholding_%28image_processing%29
+
+"""
+import matplotlib
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.filters import thresholding
+
+img = data.page()
+
+# Here, we specify a radius for local thresholding algorithm.
+# If it is not specified, only global algorithms are called.
+fig, ax = thresholding.try_all_threshold(img, radius=20,
+                                         figsize=(10, 8), verbose=False)
+plt.show()
+
+######################################################################
+# How to apply a threshold?
```

Comment:
I'm not really sure, but this probably should be removed from this example and replaced with a link to the other one. So to avoid duplicates and separate "novice"/"advanced" stuff on thresholding.